docs/ufp.md

```diff
@@ -0,0 +1,25 @@
+# Ultimaker Format Package (UFP)
```

Comment:
Overall the documentation is very marginal. I'm missing things like:

* The fact that it is based on the Open Packaging Convention
* More elaborate goal of the format (combine tool path + preview images + metadata)
* Backwards compatibility policy
* Versioning policy
* Implementation details in the libCharon library
* The plan to make this the default filetype for all UM printers
* Why it is different than 3MF (which could also have been an option, but we decided to not add the original 3D file which is a 3MF requirement)
* The plan to make it forward-compatible with 3MF if we ever decide to add the original 3D model anyways (for example for cloud based re-slicing)
* What is this Ultimaker G-code flavour?
* The allowed file type and size for the thumbnail
* Future support for multiple thumbnails (this was in the original spec iirc)
* A representative example?
* Probably some more things...